ci/requirements-3.5_DOC.pip

```diff
@@ -0,0 +1 @@
+cssdecl
```

Comment:
yes that is fine. note that you *might* want to create a conda-forge package for this at some point (otherwise you can't include it directly in a dependecy tree as they don't support pip install directly).